src/main/java/Duke.java

```diff
@@ -1,10 +1,158 @@
+                }
+                else {
+                    String deadlineTiming = taskContent.split("/by ")[1];
+                    taskFinal = taskContent.split("/")[0] + " (by:" + deadlineTiming + ")";
+                    createTask(taskFinal);
+                }
+            }
+            catch (IndexOutOfBoundsException e) {
+                System.out.println("Cannot parse deadline start date!\n"
+                + "Format: deadline [task] /by [time]");
+            }
+            break;
+
+        case "event":
+            try {
+                //empty event exception
+                String eventContent = taskContent.split("/from ")[0];
+                if (eventContent.trim().isEmpty()) {
+                    throw new EmptyTaskException();
+                }
+                else {
+                    String eventTiming = taskContent.split("/from ")[1];
+                    String eventFrom = eventTiming.split("/to ")[0];
+                    String eventTo = eventTiming.split("/to ")[1];
+                    taskFinal = taskContent.split("/")[0] + " (from: " + eventFrom + "to: " + eventTo + ")";
+                    createTask(taskFinal);
+                }
+            }
+            catch (IndexOutOfBoundsException e) {
+                System.out.println("Cannot parse event start/end date!\n"
+                +  "Format: event [task] /from [start time] /to [end time]");
+            }
+            break;
+
+        default:
+            if (taskContent.trim().isEmpty()) {
+                throw new EmptyTaskException();
+            }
+            else {
+                createTask(taskContent);
+            }
+        }
+    }
+
+    //show current tasks
+    public static void showTasks() {
+        System.out.println(NEW_LINE);
+        System.out.println("Your current tasks");
+        for (int i = 0; i < taskLength; i++){
+            System.out.print((i + 1) + ". ");
+            list[i].showTask();
+            //System.out.println(list[i].classSpecifics());
+        }
+        System.out.print(NEW_LINE);
+    }
+
+    public static void mark(int taskNumber) throws DuplicateMarkException {
+        if (list[taskNumber - 1].isDone) {
+            throw new DuplicateMarkException();
+        }
+        else {
+            list[taskNumber - 1].markAsDone();
+            System.out.println("Good job! I've marked task " + taskNumber + " as done");
+        }
+    }
+
+    public static void unMark(int taskNumber){
+        System.out.println("I've marked task " + taskNumber + " as not done");
+        list[taskNumber - 1].markNotDone();
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String chat_name = "Sigma";
```

Comment:
You should use camel form to name a normal variable, only use all caps and underscore when it is a constant.